src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,195 @@
+
+  </Step>
+
+  <Step>
+    ## Check out some dashboards
+
+    Explore your environment in real time with some out-of-the-box UI views. You'll see data in the **Infrastructure**, **APM**, and **Logs** UI pages. 
+
+    Learn how to [get started with dasboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/).
+  </Step>
+
+  <Step>
+    ## Add more data
+
+    If you need data from other sources, check our [quickstarts](https://newrelic.com/instant-observability). These quickstarts include step-by-step instructions to guide you through the setup process. You'll be able to monitor your systems in no time. With our quickstarts, monitoring becomes effortless.
+  </Step>
+
+  <Step>
+    ## Set up alerting
+
+    You can manage your <InlinePopover type="alerts" />, [create](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/) new ones, configure your [notifications](/docs/alerts-applied-intelligence/notifications/intro-notifications/), identify [anomalies](/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence/), configure [correlation policies](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions), and more. 
+
+    Some of the [quickstarts](https://newrelic.com/instant-observability) include pre-configured alerts that save you time setting up monitoring tools.
+
+    You can also check [our tutorial](/docs/tutorial-create-alerts/create-new-relic-alerts/) on getting started with New Relic alerts.
+    
+  </Step>
+
+  <Step>
+
+    ## Add users
+
+    You can add the rest of your team to New Relic. Use the **User management** UI to [add users](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks#where). See [User type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) to learn more about the different types of users. You can also check [our tutorial](/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial) on adding users and managing user access.
+  </Step>
+</Steps>
+
+
+
+# Capabilities [#capabities]
+
+Our platform offers a comprehensive suite of capabilities to help organizations monitor, troubleshoot, and optimize the performance of their digital environments. Here's an explanation of New Relic's key capabilities. If you want to learn more, explore our [tutorials](/docs/new-relic-solutions/tutorial-landing-page/) pages. 
+
+
+## Alerts & AI [#alerts]
+
+Alerts help you identify issues in your applications and code. They show you how these issues affect your users. When you receive notifications of your alerts, you can stop problems and prevent widespread user disruptions. Being aware of issues early can lead to better user experiences.
```

Comment:
I think the things that are important to mention about alerts:
1. granular alerting
2. many options for how you're alerted (email, slack, pager duty, etc)
3. anything you can create a dashboard from you can alert on
4. You can easily turn alerts on and off as needed, during maintenance downtime, for example